docs/en/Clients-Rest.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: global
+title: REST API
+nickname: REST API
+group: Clients
+priority: 3
+---
+
+For portability with other languages, the Alluxio native API is also accessible via an HTTP proxy in
+the form of a REST API.
+
+The REST API documentation is generated as part of Alluxio build and accessible through
+`${ALLUXIO_HOME}/core/server/proxy/target/miredot/index.html`. In particular, the `paths`
```

Comment:
not sure if we need to go down to this level of details, especially we can put the rest API link here